src/nyc_trees/apps/mail/libs.py

```diff
@@ -54,7 +54,6 @@ def send_to(user, message_type, *args, **kwargs):
         'from_email': from_email,
         'subject': subject,
         'body_text': body_text,
-        'sent_at': now()
```

Comment:
Thanks... been kind of ignoring Jenkins failed/unstable build status since there's been so many lately.